WORKSPACE.bazel

```diff
@@ -41,7 +41,7 @@ SNAKEYAML_VERSION = "1.33"
 
 YDB_PROTOAPI_VERSION = "1.6.4"
 
-YDB_SDK_VERSION = "2.3.7"
+YDB_SDK_VERSION = "2.3.9"
```

Comment:
This is not enough! You also must change `<ydb.sdk.version>` property in `pom.xml`!

Reply:
updated